### ISSUE TITLE
Failure to install package from local tar.bz2 file path

### ISSUE THREAD

**Issue by Boyazhou** [conda/conda#4072]
Current conda install:

```
               platform : win-64
          conda version : 4.2.13
       conda is private : False
      conda-env version : 4.2.13
    conda-build version : 1.3.5
         python version : 2.7.7.final.0
       requests version : 2.12.3
       root environment : C:\Anaconda  (writable)
    default environment : C:\Anaconda
       envs directories : C:\Anaconda\envs
          package cache : C:\Anaconda\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : None
           offline mode : False

```

I build my own conda package, and generated my pack.tar.bz2 file. I was trying to install this package on another machine from the pack.tar.bz2 file.

I ran `conda install path-to-pack.tar.bz2` 

I got below exception:

    `$ C:\Anaconda\Scripts\conda-script.py install ./bin/pack.tar.bz2`
    
        Traceback (most recent call last):
          File "C:\Anaconda\lib\site-packages\conda\exceptions.py", line 479, in con
    da_exception_handler
            return_value = func(*args, **kwargs)
          File "C:\Anaconda\lib\site-packages\conda\cli\main.py", line 145, in _main
    
            exit_code = args.func(args, p)
          File "C:\Anaconda\lib\site-packages\conda\cli\main_install.py", line 80, i
    n execute
            install(args, parser, 'install')
          File "C:\Anaconda\lib\site-packages\conda\cli\install.py", line 209, in in
    stall
            explicit(args.packages, prefix, verbose=not context.quiet)
          File "C:\Anaconda\lib\site-packages\conda\misc.py", line 175, in explicit
            index.update(get_index(**index_args))
          File "C:\Anaconda\lib\site-packages\conda\api.py", line 24, in get_index
            index = fetch_index(channel_urls, use_cache=use_cache, unknown=unknown)
          File "C:\Anaconda\lib\site-packages\conda\fetch.py", line 295, in fetch_in
    dex
            for url in urls]
          File "C:\Anaconda\lib\site-packages\conda\fetch.py", line 75, in func
            res = f(*args, **kwargs)
          File "C:\Anaconda\lib\site-packages\conda\fetch.py", line 117, in fetch_re
    podata
            timeout=(6.1, 60))
          File "C:\Anaconda\lib\site-packages\requests\sessions.py", line 501, in ge
    t
            return self.request('GET', url, **kwargs)
          File "C:\Anaconda\lib\site-packages\requests\sessions.py", line 488, in re
    quest
            resp = self.send(prep, **send_kwargs)
          File "C:\Anaconda\lib\site-packages\requests\sessions.py", line 641, in se
    nd
            r.content
          File "C:\Anaconda\lib\site-packages\requests\models.py", line 781, in cont
    ent
            self._content = bytes().join(self.iter_content(CONTENT_CHUNK_SIZE)) or b
    ytes()
          File "C:\Anaconda\lib\site-packages\requests\models.py", line 714, in gene
    rate
            chunk = self.raw.read(chunk_size)
        AttributeError: 'exceptions.WindowsError' object has no attribute 'read'

Please let me know what can be a fix.

**Comment by nehaljwani** [comment]
@Boyazhou 
Does `$ conda install absolute-path-to-pack.tar.bz2` work for you?

**Comment by marscher** [comment]
I'm getting the same error but for installing remote packages:
https://ci.appveyor.com/project/rmcgibbo/conda-recipes/build/1.0.1266/job/rdieyg6m1uvhn3i8#L134

**Comment by kalefranz** [comment]
fixed in later versions of 4.2
